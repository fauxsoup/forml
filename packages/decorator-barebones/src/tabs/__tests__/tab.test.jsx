import Tab from '../tab';
import React from 'react';
import { render } from '@testing-library/react';

describe('renders', function () {
    let form;
    beforeEach(function () {
        form = { title: 'title' };
    });
    test('its children', function () {
        const { container } = render(
            <Tab form={form}>
                <div id="test">test</div>
            </Tab>
        );

        expect(container).toMatchSnapshot();
        expect(container.querySelector('button')).not.toBeNull();
        expect(container.querySelector('button').textContent).toBe('title');
    });
});
