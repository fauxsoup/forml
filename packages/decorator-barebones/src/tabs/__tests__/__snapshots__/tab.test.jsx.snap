// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders its children 1`] = `
<div>
  <button>
    title
  </button>
</div>
`;
